src/scheduler/MpiWorld.cpp

```diff
@@ -589,15 +589,13 @@ void MpiWorld::send(int sendRank,
     // If the message is set and recording on, track we have sent this message
     if (thisRankMsg != nullptr && thisRankMsg->recordexecgraph()) {
         faabric::util::exec_graph::incrementCounter(
-          *thisRankMsg,
-          faabric::util::exec_graph::mpiMsgCountPrefix +
-            std::to_string(recvRank));
+          *thisRankMsg, MPI_MSG_COUNT_PREFIX + std::to_string(recvRank));
 
         // Work out the message type breakdown
         faabric::util::exec_graph::incrementCounter(
           *thisRankMsg,
           fmt::format("{}-{}-{}",
-                      faabric::util::exec_graph::mpiMsgTypeCountPrefix,
+                      MPI_MSGTYPE_COUNT_PREFIX,
```

Comment:
Does this string format then result in a double hyphen at the beginning (as the prefix already has a hyphen)?

Reply:
For the `MSGTYPE_COUNT` we have to provide two values, so the constant actually hasn't got a trailing hyphen. I will change it to make them both consistent.